src/decide.ts

```diff
@@ -102,6 +104,48 @@ export function compareFileTreesIgnoringNames(
 	);
 }
 
+export function compareFileTreesPartialIgnoringNames(
+	candidate: Metafile,
+	searchee: Searchee
+): boolean {
+	const { fuzzySizeThreshold } = getRuntimeConfig();
+	let matchedSizes = 0;
+	for (const candidateFile of candidate.files) {
+		let matchedSearcheeFiles = searchee.files.filter(
+			(searcheeFile) => searcheeFile.length === candidateFile.length
+		);
+		if (matchedSearcheeFiles.length) {
+			matchedSizes += candidateFile.length;
+		}
+	}
+	return matchedSizes / candidate.length >= 1-fuzzySizeThreshold;
```

Comment:
this code needs to be run through prettier
`npx prettier --write "src/**/*.ts"`